requirePromise.js

```diff
@@ -1,7 +1,7 @@
 'use strict';
 
 module.exports = function requirePromise() {
-	if (typeof Promise !== 'function') {
+	if (typeof (Promise || global.Promise || window.Promise) !== 'function') {
```

Comment:
Um, they certainly should be - are you using some kind of bundler that changes the semantics of the module? Can you provide a jsfiddle that reproduces that?